include/boost/spirit/home/x3/support/traits/container_traits.hpp

```diff
@@ -122,15 +122,15 @@ namespace boost { namespace spirit { namespace x3 { namespace traits
         template <typename T>
         static bool call(Container& c, T&& val)
         {
-            c.insert(c.end(), std::move(val));
+            c.insert(c.end(), static_cast<T&&>(val));
             return true;
         }
     };
 
     template <typename Container, typename T>
     inline bool push_back(Container& c, T&& val)
     {
-        return push_back_container<Container>::call(c, std::move(val));
+        return push_back_container<Container>::call(c, static_cast<T&&>(val));
```

Comment:
What's the reason behind such changes? Reducing use of standard library (compile time)?

Reply:
Removed surprising move of lvalue. And yes, `std::forward` is not zero cost in terms of compile time (also a surprising thing).